src/main/java/com/epam/esm/controller/GiftCertificatesController.java

```diff
@@ -100,12 +108,12 @@ public GiftCertificatesController(GiftCertificateService service) {
      * @param id - id of {@link GiftCertificate} object to be updated.
      * @param giftCertificate - new data for updating an {@link GiftCertificate} object to be updated.
      * @return {@link ResponseEntity} contained both {@link HttpStatus} status and updated {@link GiftCertificate} object.
-     * @throws BadEntityException in case if invalid data passed in request to upadte a {@link GiftCertificate} object.
-     * @throws EntityNotExistsException in case if nothing found with searched id.
      */
     @PatchMapping(value = "/{id}", produces = JSON)
```

Comment:
Do you need to specify this `produces = JSON`?

Reply:
No need to explicitly write this